src/_guides/language/language-tour.md

```diff
@@ -720,6 +721,83 @@ information about lists, see [Generics](#generics) and
 [Collections](/guides/libraries/library-tour#collections).
 
 
+### Sets
+
+A set in Dart is an unordered collection of unique items.
+Dart support for sets is provided by set literals and the [Set][] type.
+
+<aside class="alert alert-info" markdown="1">
+  **Version note:** Although the Set _type_ has always been a core part of Dart,
+  set _literals_ were introduced in Dart 2.2.
+</aside>
+
+Here is a simple Dart set, created using a set literal:
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (set-literal)"?>
+{% prettify dart %}
+var halogens = {'fluorine', 'chlorine', 'bromine', 'iodine', 'astatine'};
+{% endprettify %}
+
+<aside class="alert alert-info" markdown="1">
+  **Note:**
+  The analyzer infers that `halogens` has the type
+  `Set<String>`. If you try to add the wrong type of value
+  to the set, the analyzer or runtime raises an error.
+  For more information, read about
+  [type inference.](/guides/language/sound-dart#type-inference)
+</aside>
+
+To create an empty set, use `{}` preceded by a generic type annotation,
+or assign `{}` to a variable of type `Set`:
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (set-vs-map)"?>
+{% prettify dart %}
+var names = <String>{};
+// Set<String> names = {}; // This works, too.
+// var names = {}; // Creates a map, not a set.
+{% endprettify %}
+
+<aside class="alert alert-info" markdown="1">
+  **Set or map?**
+  The syntax for map literals is similar to that for set literals.
+  Because map literals came first, `{}` defaults to the `Map` type.
+  If you forget the type annotation on `{}` or the variable it's assigned to,
+  then Dart creates an object of type `Map<dynamic, dynamic>`.
+</aside>
+
+Add items to an existing set using the `add()` or `addAll()` methods:
+
+<?code-excerpt "misc/lib/language_tour/built_in_types.dart (set-add-items)"?>
+{% prettify dart %}
+var elements = <String>{};
+elements.add('fluorine');
+elements.addAll(halogens);
+{% endprettify %}
+
+Use `.length` to get the number of items in the set:
+
+<?code-excerpt "misc/test/language_tour/built_in_types_test.dart (set-length)"?>
+{% prettify dart %}
+var elements = <String>{};
+elements.add('fluorine');
+elements.addAll(halogens);
+assert(elements.length == 5);
+{% endprettify %}
+
+To create a set that's a compile-time constant,
+add `const` before the set literal:
```

Comment:
... or just use it in a constant context

....

const colorGroups = { "warm": {"red", "yellow", "orange"}, "cold": {"green", "blue", "purple"}};

Reply:
Didn't get this done

Reply:
This creates a compile-time constant without writing `const` before the literal (well, not *just* before the literal).